reana_job_controller/kubernetes_job_manager.py

```diff
@@ -314,6 +320,51 @@ def _add_krb5_init_container(self, secrets_volume_mount):
         self.job['spec']['template']['spec']['initContainers'].append(
             krb5_container)
 
+    def _add_proxy_init_container(self, secrets_volume_mount):
+        """Add  sidecar container for a job."""
+        ticket_cache_volume = {
+            'name': 'proxy-cache',
+            'emptyDir': {}
+        }
+        volume_mounts = [
+            {
+                'name': ticket_cache_volume['name'],
+                'mountPath': current_app.config['PROXY_CERT_CACHE_LOCATION']
+            }
+        ]
+
+        proxy_pass = os.environ.get('CERN_PROXYPASS')
+        proxy_file_path = os.path.join(
+                               current_app.config['PROXY_CERT_CACHE_LOCATION'],
+                               current_app.config['PROXY_CERT_CACHE_FILENAME']
+                           )
+
+        proxy_container = {
+            'image': current_app.config['PROXY_CONTAINER_IMAGE'],
+            'command': ['/bin/bash'],
+            'args': ['-c', 'echo {} | base64 -d | voms-proxy-init \
+                     --voms cms --key $(readlink -f /etc/reana/secrets/userkey.pem) \
+                     --cert $(readlink -f /etc/reana/secrets/usercert.pem) \
+                     --pwstdin --out {}; \
+                     chown 1000 {}'.format(proxy_pass, proxy_file_path, proxy_file_path)],
```

Comment:
I recently changed this to `chown {} {}'.format(proxy_pass, proxy_file_path, self.kubernetes_uid, proxy_file_path)],` but I see I forgot to push it before creating PR